build/build_content/clone.sl

```diff
@@ -0,0 +1,32 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```

Comment:
it's a very hacky way of cloning.
I think we should find a better way for git commands..